src/components/monumentCard/index.ts

```diff
@@ -0,0 +1,6 @@
+export const createMainContent = (): HTMLElement => {
+  const mainContent = document.createElement("main");
```

Comment:
Si la carpeta se llama monumentCard, entiendo que habrá una card, porque es una etiqueta main?

Reply:
Tenéis razón, lo cambio en esta pull requests